### DISCUSSION TITLE
Mods were not found in the ArmA directory

### DISCUSSION THREAD

**Discussion by VComrade** [Foxlider/FASTER#135] [Q&A] [answered]
Hello, maybe someone might help me?
I try to setup server, but when I click "save profile" in FASTER it says "30 mods were not found in the ArmA directory".
Any idea why? I did everything that guide says.

**Comment by jupster** [comment]
What is your servers pathings?

**Reply by VComrade** [reply]
"E:\Arma server" if that's what you mean?

**Reply by jupster** [reply]
Sorry what is the destination to your 

arma root
mods 
directory

**Reply by VComrade** [reply]
"E:\ARMA_mods" for server and ArmA's default is "D:\Games\Steam Games\steamapps\workshop\content"

**Reply by jupster** [reply]
Are you downloading your mods through steamCMD

**Reply by jupster** [reply, marked as answer]
C:\Games\ArmA3\A3 (root)
C:\Games\ArmA3\A3Master\Server_1 (mod path)
C:\Games\ArmA3\A3Master\Mod_Staging (directory path)


My pathing

**Reply by VComrade** [reply]
I downloaded mods through FASTER and server files through steamCMD

**Reply by jupster** [reply]
Right that isnt really needed but thats fine.

If you ref my pathing you need a similar setup. 

One is root
One is mods
One is deployment

**Reply by VComrade** [reply]
What is "deployment" path?
Do I need to make my pathing exactly as yours?

**Reply by VComrade** [reply]
Do I need to have my server mods folder inside of the directory that my Arma game is installed in?

**Reply by schnookiecakes** [reply]
I am having the same issue as well.

mod staging: C:\Program Files (x86)\Steam\steamapps\workshop\content\107410

install dir: C:\Program Files (x86)\Steam\steamapps\common\Arma 3 Server

deployment dir: C:\Program Files (x86)\Steam\steamapps\common\Arma 3 Server\mods

**Reply by jupster** [reply]
> I am having the same issue as well.
> 
> mod staging: C:\Program Files (x86)\Steam\steamapps\workshop\content\107410
> 
> install dir: C:\Program Files (x86)\Steam\steamapps\common\Arma 3 Server
> 
> deployment dir: C:\Program Files (x86)\Steam\steamapps\common\Arma 3 Server\mods

You might need to check out this page https://github.com/Foxlider/FASTER/wiki/Use-Faster-with-Steam
